### DISCUSSION TITLE
Windows Startbutton and Searchbutton don't work any more

### DISCUSSION THREAD

**Discussion by h3rm** [valinet/ExplorerPatcher#330] [General]
hi! i just configured everything as you wrote in the readme but after restarting the explorer with taskkill /f /im explorer.exe the start- and searchbuttons in the taskbar don't work and more.

Edition	Windows 11 Pro
Version	21H2
Installiert am	‎06.‎10.‎2021
Betriebssystembuild	22000.318
Leistung	Windows Feature Experience Pack 1000.22000.318.0

https://i.imgur.com/mmorCpz.png
https://i.imgur.com/Dk4z2qH.png
https://i.imgur.com/0zccc2M.png
https://i.imgur.com/ff7CIIT.png
https://i.imgur.com/HZQNSDn.png
https://i.imgur.com/F4w1V8A.png
https://i.imgur.com/caqQ1fc.png

also the mouse mouse cursor is flickering (round circle/loading symbol) every 1-2 seconds asince about 1.5 hours and sometimes the task bar is gone for a few seconds. After 1 hour of waiting i tried to restart but there is still the same problem..

is it really possible that this such a long time? i never saw any countdown for downloading the symbols, but some are still missing:
https://i.imgur.com/D4GzdTY.png

please help! i have to work in 6 hours :D 

eventlog:
https://i.imgur.com/2t8uh4P.png
https://i.imgur.com/LT34WMQ.png

**Comment by valinet** [comment]
General directions for troubleshooting: https://github.com/valinet/ExplorerPatcher/issues/320#issuecomment-966469498

Thanks
